src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -43,29 +42,82 @@
     from argilla.client.workspaces import Workspace
 
 
-class RemoteFeedbackRecords(RemoteFeedbackRecordsBase):
-    def __init__(self, dataset: "RemoteFeedbackDataset") -> None:
-        super().__init__(dataset=dataset)
+class RemoteFeedbackRecords(ArgillaRecordsMixin):
+    def __init__(
+        self,
+        dataset: "RemoteFeedbackDataset",
+        response_status: Optional[Union["FeedbackResponseStatusFilter", List["FeedbackResponseStatusFilter"]]] = None,
+        metadata_filters: Optional[Union["MetadataFilters", List["MetadataFilters"]]] = None,
+        sort_by: Optional[List[SortBy]] = None,
```

Comment:
@frascuchon can we once again add a sort_by  datetime here?

Reply:
inserted_at I mean

Reply:
Yes, The SortBy.fields is or str or `RecordSortBy.inserted_at` and `RecordSortBy.inserted_at` which can be configure by string directly:
```python
ds.sort_by([SortBy(field="inserted_at", order="desc")]
```